gpflow/base.py

```diff
@@ -324,9 +334,32 @@ def _run_op(a, *args, **kwargs):
 """
 
 
-def _cast_to_dtype(value: VariableData, dtype: Optional[DType] = None) -> tf.Tensor:
+class Module(tf.Module):
+    @property
+    def parameters(self) -> Tuple[Parameter, ...]:
```

Comment:
does this syntax mean ... a tuple of Parameters? or one Parameter, and unkown stuff?

Reply:
a tuple of parameters

Reply:
i think

Reply:
https://docs.python.org/3/library/typing.html#typing.Tuple